src/docker/Dockerfile

```diff
@@ -32,6 +32,7 @@ LABEL Description="This is the image for aether.jcabi.com" Vendor="jcabi.com" Ve
 WORKDIR /tmp
 
 ENV DEBIAN_FRONTEND=noninteractive
+ENV OPENJDK_VERSION 1.7.0_75
```

Comment:
@amihaiemil this would be fine as well, it is very unlikely that new Java 7 version introduces breaking change :)